windows-mixin/dashboards/windows_exporter.libsonnet

```diff
@@ -0,0 +1,422 @@
+      .addPanel(errorService, gridPos={ x: 8, y: 16, w: 8, h: 4 })
+      .addPanel(diskUsage, gridPos={ x: 16, y: 16, w: 8, h: 4 })
+      .addPanel(diskIO, gridPos={ x: 0, y: 20, w: 8, h: 8 })
+      .addPanel(networkUsage, gridPos={ x: 8, y: 20, w: 8, h: 8 })
+      .addPanel(iisConnections, gridPos={ x: 16, y: 20, w: 8, h: 8 }),
+
+  },
+
+  local integration_status_panel =
+    grafana.statPanel.new(
+      'Integration Status',
+      datasource='$prometheus_datasource',
+      colorMode='background',
+      graphMode='none',
+      noValue='No Data',
+      reducerFunction='lastNotNull'
+    )
+    .addMappings(
+      [
+        {
+          options: {
+            from: 1,
+            result: {
+              color: 'green',
+              index: 0,
+              text: 'Agent Configured - Sending Metrics',
+            },
+            to: 10000000000000,
+          },
+          type: 'range',
+        },
+        {
+          options: {
+            from: 0,
+            result: {
+              color: 'red',
+              index: 1,
+              text: 'No Data',
+            },
+            to: 0,
+          },
+          type: 'range',
+        },
+      ]
+    )
+    .addTarget(
+      grafana.prometheus.target('sum(windows_cpu_time_total{' + host_matcher + ',mode="idle"})')
+    ),
+
+  local latest_metric_panel =
+    grafana.statPanel.new(
+      'Latest Metric Received',
+      datasource='$prometheus_datasource',
+      colorMode='background',
+      fields='Time',
+      graphMode='none',
+      noValue='No Data',
+      reducerFunction='lastNotNull'
+    )
+    .addTarget(
+      grafana.prometheus.target('sum(windows_cpu_time_total{' + host_matcher + ',mode="idle"})')
+    ),
+
+  local usageTable =
+    win.wintable('Usage', '$prometheus_datasource')
+    .addQuery('windows_os_info{' + host_matcher + '} * on(instance) group_right(product) windows_cs_hostname', 'group', 'group')
+    .addQuery('100 - (avg by (instance) (rate(windows_cpu_time_total{' + host_matcher + ',mode="idle"}[$__rate_interval])) * 100)', 'CPU Usage %', 'cpuusage')
+    .addQuery('time() - windows_system_system_up_time{' + host_matcher + '}', 'Uptime', 'uptime')
+    .addQuery('windows_cs_logical_processors{' + host_matcher + '} - 0', 'CPUs', 'cpus')
+    .addQuery('windows_cs_physical_memory_bytes{' + host_matcher + '} - 0', 'Memory', 'memory')
+    .addQuery('100 - 100 * windows_os_physical_memory_free_bytes{' + host_matcher + '} / windows_cs_physical_memory_bytes{' + host_matcher + '}', 'Memory Used', 'memoryused')
+    .addQuery('(windows_logical_disk_free_bytes{' + host_matcher + ',volume=~"C:"}/windows_logical_disk_size_bytes{' + host_matcher + ',volume=~"C:"}) * 100', 'C:\\ Free %', 'cfree')
+    .hideColumn('Time')
+    .hideColumn('domain')
+    .hideColumn('fqdn')
+    .hideColumn('job')
+    .hideColumn('agent_hostname')
+    .hideColumn('Value #group')
+    .hideColumn('instance')
+    .renameColumn('Value #cpuusage', 'CPU Usage %')
```

Comment:
can we rename product and hostname columns as well?
hosname-> Hostname
product -> Product

Reply:
Renamed - Pushing changes in latest commit